src/demo/index.php

```diff
@@ -106,6 +106,8 @@ function gtag() {
                     <label>✕</label>
                     <input class="param inline" type="number" id="height" name="height" alt="Height (px)" placeholder="50" value="50">
                 </span>
+
+                <input type="button" value="Reset" onclick="preview.reset();">
```

Comment:
Thanks! Just one last thing :+1: 

Currently the button looks like this:

![image](https://user-images.githubusercontent.com/20955511/201512480-46e1261f-8d13-4e2a-b244-1fad82711f58.png)

If we add the "btn" class (`class="btn"`) like the other buttons have, it will apply the same styling so it will look similar to the others.

![image](https://user-images.githubusercontent.com/20955511/201512484-96bfa48e-cee4-4c56-a45d-a074e7b23e9c.png)

```suggestion
                <input type="button" class="btn" value="Reset" onclick="preview.reset();">
```